scenarios/centos-9/ci.yml

```diff
@@ -5,11 +5,25 @@ cifmw_installyamls_repos: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-o
 cifmw_use_libvirt: true
 cifmw_use_crc: true
 
+pre_infra:
```

Comment:
is it really needed ?

Reply:
Yes, this is not done anymore by the base-crc job, that called the download_tools target here: https://github.com/rdo-infra/review.rdoproject.org-config/blob/master/playbooks/crc/openstack.yaml#L11-L14